components/formats-common/src/loci/common/IniParser.java

```diff
@@ -177,7 +177,10 @@ public IniList parseINI(BufferedReader in)
 
       // parse key/value pair
       int equals = line.indexOf("=");
-      if (equals < 0) throw new IOException(no + ": bad line");
+      if (equals < 0) {
+        LOGGER.debug("Ignoring line %d", no);
```

Comment:
Doh, thanks. I meant to double check that syntax before pushing but forgot. Fixed.